src/hotspot/share/gc/g1/g1MonitoringSupport.cpp

```diff
@@ -145,8 +137,8 @@ G1MonitoringSupport::G1MonitoringSupport(G1CollectedHeap* g1h) :
   // and used.
   _old_space_counters = new HSpaceCounters(_old_gen_counters->name_space(),
     "space", 0 /* ordinal */,
-    pad_capacity(g1h->max_capacity()) /* max_capacity */,
-    pad_capacity(_old_gen_committed) /* init_capacity */);
+    g1h->max_capacity() /* max_capacity */,
+    _old_gen_committed /* init_capacity */);
```

Comment:
[pre-existing] The indentation of all these HSpaceCounters constructor calls is pretty unusual.